src/demo.tsx

```diff
@@ -112,19 +133,30 @@ const messages: MessageProps[] = [
 	},
 	{
 		message: {
-			source: "bot",
-			text: "This messaged is with previous with disableHeader prop",
-			avatarName: "Cognigy",
-		},
-		disableHeader: true,
-	},
-	{
-		message: image as IMessage,
```

Comment:
why you removed the fixture here? is much less chaotic using directly the fixture when possible

Reply:
mistook by conflict resolution, fixed